docs/blogs/2023/2023-06-07-fast-conformer.md

```diff
@@ -0,0 +1,210 @@
+
+## ASR Results
+
+Below, we list some of our results on Fast Conformer on LibriSpeech test-other. We compare against the original Conformer and other recent efficient architectures. We compare against the [Efficient Conformer from Burchi2021](https://arxiv.org/abs/2109.01163) which uses a progressive downsampling of the Conformer architecture. We also compare against [Kim2022 SqueezeFormer](https://arxiv.org/abs/2206.00888) which uses a U-Net like architecture to progressively downsample the input and upsample it to 40 ms resolution prior to applying the decoder. 
+
+We find that Fast Conformer is able to achieve the same accuracy as the Conformer while being 2.8x faster and using fewer parameters. 
+
+*Fast Conformer-Large with CTC and RNNT decoders trained on Librispeech. Greedy WER (%) was measured on Librispeech test-other.  The number of parameters  (M) and compute (Multiply-Acc, GMAC) are shown for encoder only.*
+
+| Encoder                     	|   WER, %   	| Params, 	| Compute, 	|
+|-----------------------------	|:----------:	|:-------:	|:--------:	|
+|                             	| test-other 	|    M    	|   GMACS  	|
+|         RNNT decoder        	|            	|         	|          	|
+| Conformer                   	|    5.19    	|   115   	|   143.2  	|
+| Fast Conformer              	|    4.99    	|   109   	|   48.7   	|
+|         CTC decoder         	|            	|         	|          	|
+| Conformer                   	|    5.74    	|   121   	|   149.2  	|
+| Eff. Conformer [Burchi2021] 	|    5.79    	|   125   	|   101.3  	|
+| SqeezeFormer [Kim2022]      	|    6.05    	|   125   	|   91.0   	|
+| Fast Conformer              	|    6.00    	|   115   	|   51.5   	|
+
+## Speech Translation Results
+
+We also evaluate Fast Conformer on the IWSLT 2014 German-English speech translation task. We find that Fast Conformer is able to achieve the same accuracy as the Conformer while being upto 1.8x faster and using fewer parameters.
+
+Our models have been trained on all available datasets from IWSLT22 competition which corresponds to 4k hours of speech. Some of the datasets did not contain German translations, so we generated them ourselves with text-to-text machine translation model trained on WMT21 and in-domain finetuned on Must-C v2.
+
+*Speech Translation, MUST-C V2 tst-COMMON dataset. SacreBLEU, total inference time, and relative inference speed-up were measured with a batch size $32$ for two speech translation models with Conformer-based encoder and either RNNT, or Transformer decoder.*
+
+| Encoder             	| BLEU 	| Time, sec 	| Speed-up 	|
+|---------------------	|:----:	|:---------:	|:--------:	|
+| Transformer decoder 	|      	|           	|          	|
```

Comment:
sam here regarding table parsing. 
In top row, Time, sec -> Time (sec)

Reply:
Table parsing will be different between latex and markdown. Made the headers bold and updated Time.